### ISSUE TITLE
Make "GET _license/feature_usage" API per cluster but not per node

### ISSUE THREAD

**Issue by kunisen** [elastic/elasticsearch#92464]
### Description

Per an internal sync, we observed the following things:
- The way this API is implemented in Elasticsearch is on a per-node basis. That is, the API only returns features that are being used on the node that responds to the API call.
- However, our cloud UI is calling this API against _all_ Elasticsearch cluster and integrating the result into UI.

(Please see `https://github.com/elastic/<internal cloud related repository>/issues/98111` for more detail)

The discrepancy caused this API is not very helpful in the cloud, because from a cloud API perspective, it's hard for us to specify a particular node on which we could run the API.

That said, it would be great if the ES team can enhance this API, to gather the feature usage from a cluster level.
In other words, make it return the same results from any node, regardless of which node this API is executed.

Please correct me if I labeled the wrong team, and thanks!

cc @jakommo @laraMorenoIgle
